diesel_infer_schema/src/information_schema.rs

```diff
@@ -77,6 +89,12 @@ pub fn determine_column_type(attr: &ColumnInformation) -> Result<ColumnType, Box
         &attr.type_name
     };
 
+    let tpe = if let Some(idx) = tpe.find('(') {
```

Comment:
I'm going to pull out separate `determine_column_type` functions for MySQL and PG, as there's several cases I want to handle. This is required to turn things like `int(11)` into `int` for now.